pkg/api/types.go

```diff
@@ -12,6 +12,30 @@ type ImageTag struct {
 	Timestamp    time.Time    `json:"timestamp"`
 	OS           OS           `json:"os,omitempty"`
 	Architecture Architecture `json:"architecture,omitempty"`
+
+	// If this is a Manifest list we need to keep them together
+	Children []*ImageTag `json:"children,omitempty"`
+}
+
+func (i *ImageTag) HasChildren() bool {
+	return len(i.Children) > 0
+}
```

Comment:
unused

Reply:
Ah yea, I added it and then didn't use 😄 👍